Major Editorial Update (#826)
ethereum-oasis-op/baseline: docs/baseline-basics/faq.md

# FAQ

## Is there a token for Baseline?

There is no Baseline token.

> The Baseline Protocol is a standardized framework, with requirements outlined in the [Baseline Protocol Standard](https://github.com/ethereum-oasis-op/baseline-standard), which is undergoing ratification to become an [OASIS Open Standard](https://www.oasis-open.org/standards/). All reference implementations do not employ a token.

Author: no

> The Baseline Protocol is a standardized framework, with requirements outlined in the [Baseline Protocol Standard](https://github.com/ethereum-oasis-op/baseline-standard), which is undergoing ratification to become an [OASIS Open Standard](https://www.oasis-open.org/standards/). All reference implementations do not employ a token.

Reviewer: Possible rephrase

```suggestion
The Baseline Protocol is a standardized framework, with requirements outlined in the [Baseline Protocol Standard](https://github.com/ethereum-oasis-op/baseline-standard), which is undergoing ratification to become an [OASIS Open Standard](https://www.oasis-open.org/standards/). No reference implementations employ a token.
```

* _**Baseline Core Devs:** Baseline Core Devs_ are the backbone of the Baseline Protocol open source community and have an active role in advancing the Baseline Protocol and/or related projects. Join the next generation of top blockchain developers leading the rise of ubiquitous sync services to state machines everywhere!
* **Outreach Team**: Outreach members communicate, advocate, and educate the world on the advantages of using and contributing to the protocol. Several working groups create enablement materials, devise industry-specific use cases, and document the significant work being done in the community.
* **Standards Team**: Members of the team work to define and contribute to the specifications of baseline compliance. The[ Standard ](https://github.com/ethereum-oasis-op/baseline-standard)is currently in the ratification process to become an official [Oasis Standard](https://www.oasis-open.org).

## I want to build with the Baseline Protocol, where do I start?

**Reference Implementations:** You can also choose to build on top of existing reference implementations.

> We recommend starting with BRI-3, this reference implementation of the core interfaces specified in the v1.0 release has been developed by individual members of the Baseline community. You can get started with BRI-1 [here](https://github.com/ethereum-oasis-op/baseline/tree/main/examples/bri-3).

Author: fixed
